### PR TITLE
Pie style at initialization and canvas mouseover

### PR DESCRIPTION
**Pull request by riltim**
There is no object field for mouseover, and _containsPoint was not yet implemented (for preferCanvas compatibility). Also, the Pie slices style options were not set at initialization.

### CODE REVIEW

File: Semicircle.js [hunk 1]
```diff
@@ -89,6 +89,25 @@
                 !(startAngle === 0 && stopAngle > 359) &&
                 !(startAngle == stopAngle)
             );
+        },
+        _containsPoint: function (p) {
+            function normalize (angle) {
+                var newAngle = angle;
```
Comment (jieter) [inline Semicircle.js]: Why declare a new variable here?

File: Semicircle.js [hunk 2]
```diff
@@ -89,6 +89,25 @@
...
+                while (newAngle <= -Math.PI)
```
Comment (jieter) [inline Semicircle.js]: Please adhere to the surrounding brace style: always use braces for `while`/`if` statements.

**Comment by riltim** [comment]
Thanks for the feedback @jieter! I changed the brace style and removed the variable, which was indeed unnecessary.

File: piechart/Pie.js [hunk 3]
```diff
@@ -133,10 +133,14 @@
                     color: this._color(i)
                 }, this.options.sliceOptions);
 
-                var slice = L.circle(this._latlng, options).addTo(this);
+                this._data[i].slice = L.circle(
+                   this._latlng,
+                   this.options.radius, 
+                   L.Util.extend({}, options, this.options.pathOptions)
```
Comment (jieter) [inline piechart/Pie.js]: @riltim Hmm, sorry I missed this before, but this seems to have 3-space indentation. Also: L.Circle now accepts `radius` as an option, no need to specify the radius explicitly as second argument.

Comment (riltim) [inline piechart/Pie.js]: @jieter It seems that I should be more careful in coding style... Sorry, fixed!

**Comment by jieter** [comment]
@riltim thanks, just releases leaflet-semicircle@1.0.6

**Pull request merged**